skill-tree/src/main/java/com/RDS/skilltree/User/UserRole.java

```diff
@@ -1,8 +1,30 @@
 package com.RDS.skilltree.User;
 
-public enum UserRole {
-    USER,
-    MEMBER,
-    SUPERUSER,
-    MAVEN
+import org.springframework.security.core.GrantedAuthority;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+public enum UserRole  {
+    USER("user"),
+    MEMBER("member"),
+    SUPERUSER("super_user");
+
+    public final String label;
+
+     UserRole(String label) {
+        this.label = label;
+    }
+    public static UserRole fromString(String text) {
+        for (  UserRole b : UserRole.values()) {
+            if (b.label.equalsIgnoreCase(text)) {
+                return b;
+            }
+        }
+        return null;
+    }
+    public static String[] getAllRoles(){
```

Comment:
getAllRoles method retrieves all labels using a stream and then converts them back to an array. This can be simplified by directly collecting the labels into an array using mapToArray(String[]::new).

Reply:
tried this but that was not fulfilling my requirement of converting 
```
UserRoles i.e., [{name: "USER", label:"user"}, name: "MEMBER", label:"member"}]
```
to 
```
["USER", "MEMBER",...]
```